spec/lcov_parser_spec.rb

```diff
@@ -18,6 +18,18 @@
     expect(second).to eq([[3, 1], [5, 1], [7, 1], [27, 0], [28, 0], [29, 0]])
   end
 
+  it 'reports a structure of source files with branch coverage' do
+    parser = described_class.parse('spec/fixtures/imagen_rb.lcov')
```

Comment:
Yep I agree. I decided to reuse `imagen_rb` since you already used it (and wrote it).
But I would say the ideal would be to use some exemplary code as input instead.

Reply:
Can you remove this files and create a new fixture file or update `spec/fixtures/sample.lcov` that's used in the spec example above?